insights/parsers/tests/test_netstat.py

```diff
@@ -528,15 +527,15 @@ def test_get_netstat_i():
     result = Netstat_I(context_wrap(NETSTAT_I)).group_by_iface
     assert len(result) == 7
     assert result["bond0"] == {
-            "MTU": "1500", "Met": "0", "RX-OK": "845265", "RX-ERR": "0",
-            "RX-DRP": "0", "RX-OVR": "0", "TX-OK": "1753", "TX-ERR": "0",
-            "TX-DRP": "0", "TX-OVR": "0", "Flg": "BMmRU"
-                }
+        "MTU": "1500", "Met": "0", "RX-OK": "845265", "RX-ERR": "0",
+        "RX-DRP": "0", "RX-OVR": "0", "TX-OK": "1753", "TX-ERR": "0",
+        "TX-DRP": "0", "TX-OVR": "0", "Flg": "BMmRU"
+    }
     assert result["eth0"] == {
-            "MTU": "1500", "Met": "0", "RX-OK": "422518", "RX-ERR": "0",
-            "RX-DRP": "0", "RX-OVR": "0", "TX-OK": "1703", "TX-ERR": "0",
-            "TX-DRP": "0", "TX-OVR": "0", "Flg": "BMsRU"
-                }
+        "MTU": "1500", "Met": "0", "RX-OK": "422518", "RX-ERR": "0",
+        "RX-DRP": "0", "RX-OVR": "0", "TX-OK": "1703", "TX-ERR": "0",
+        "TX-DRP": "0", "TX-OVR": "0", "Flg": "BMsRU"
+    }
```

Comment:
Please add corresponding explanation to the git-log message for these unrelated format update.